base/standard/src/main/java/org/openscience/cdk/fingerprint/Fingerprinter.java

```diff
@@ -394,17 +394,18 @@ private String getAtomSymbol(IAtom atom) {
      *@return       The bondSymbol value
      */
     protected String getBondSymbol(IBond bond) {
-        String bondSymbol = "";
-        if (bond.getFlag(CDKConstants.ISAROMATIC)) {
-            bondSymbol = ":";
-        } else if (bond.getOrder() == IBond.Order.SINGLE) {
-            bondSymbol = "-";
-        } else if (bond.getOrder() == IBond.Order.DOUBLE) {
-            bondSymbol = "=";
-        } else if (bond.getOrder() == IBond.Order.TRIPLE) {
-            bondSymbol = "#";
+        if (bond.isAromatic())
+            return ":";
+        switch (bond.getOrder()) {
+            case SINGLE:
+                return "-";
+            case DOUBLE:
+                return "=";
+            case TRIPLE:
+                return "#";
+            default:
+                return "";
```

Comment:
This is a nice example of maintenance... "switch" did not exist when Chris originally wrote this code now close to 20 years ago :)

BTW, is the switch seriously enough faster to notice?? Or is it just not doing the getOrder() so often?